airbyte-ci/connectors/base_images/base_images/python/v1.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+"""This module declares all the airbyte python connector base image for version 1.
+Please create a v2.py module if you want to declare a new major version.
+"""
+from __future__ import annotations
+
+from typing import Final, final
+
+import dagger
+from base_images import common, sanity_checks
+from base_images.python.common import AirbytePythonConnectorBaseImage, PythonBase
+
+
+class _1_0_0(AirbytePythonConnectorBaseImage):
+    base_base_image: Final[PythonBase] = PythonBase.PYTHON_3_9_18
+
+    changelog_entry: Final[
+        str
+    ] = "Declare our first base image version. It uses Python 3.9.18 on a Debian 11 (Bookworm) system with Pip 23.2.1 and UTC timezone."
+
+    run_previous_version_sanity_checks = False
```

Comment:
Honest opinion: I dont think this functionality is worth while and wed be better off not having direct dependencies in any way between these images

Reply:
As we discussed in our call we agreed to make the `container` property fully explicit and not call `super`. I remain align with this for readability. 
But for sanity_checks I'm not :D : 
- If we are not using inheritance in `container` we'll have to duplicate `container` declaration from a previous version to a new version. It's easy to sneak in a typo in this process (let's say a wrong poetry package version). Running sanity checks on the previous version with the container of the new version make it possible to detect this kind of error.